examples/ex5/ex5.cpp

```diff
@@ -190,6 +191,8 @@ int main(int argc, char **argv) {
       printf("------Plot %d: WriteVTU by rank : %d\n", plot_counter,
              world_rank);
       WriteVTU(argv[1], plot_counter, Time);
+      stepTime[nStep] = Time;
```

Comment:
Check nStep < nPlotSteps to prevent seg. fault

Reply:
agree